--- PyJWT-1.7.1/setup.py.orig	2018-12-06 13:28:35.000000000 +0100
+++ PyJWT-1.7.1/setup.py	2021-01-20 21:48:02.372776650 +0100
@@ -33,9 +33,9 @@
     sys.exit()
 
 tests_require = [
-    'pytest>=4.0.1,<5.0.0',
-    'pytest-cov>=2.6.0,<3.0.0',
-    'pytest-runner>=4.2,<5.0.0',
+    'pytest>=4.0.1',
+    'pytest-cov>=2.6.0',
+    'pytest-runner>=4.2',
 ]
 
 needs_pytest = set(('pytest', 'test', 'ptr')).intersection(sys.argv)
--- PyJWT-1.7.1/tests/contrib/test_algorithms.py.orig	2018-11-26 04:04:17.000000000 +0100
+++ PyJWT-1.7.1/tests/contrib/test_algorithms.py	2021-01-20 21:47:38.742904664 +0100
@@ -161,7 +161,10 @@
         with open(key_path('testkey_ec.pub'), 'r') as keyfile:
             jwt_pub_key = algo.prepare_key(keyfile.read())
 
-        result = algo.verify(jwt_message, jwt_pub_key, jwt_sig)
+        try:
+            result = algo.verify(jwt_message, jwt_pub_key, jwt_sig)
+        except: # ecdsa>=0.14 raises BadSignatureError
+            result = False
         assert not result
 
     def test_ec_verify_should_return_true_if_signature_valid(self):
